libmscore/lyrics.cpp

```diff
@@ -261,8 +261,8 @@ void Lyrics::layout()
                         Lyrics trailing(*this);
                         trailing.setPlainText(tp);
                         trailing.layout1();
-                        leftAdjust = leading.width();
-                        centerAdjust = leading.width() - trailing.width();
+                        leftAdjust = leading.width() + symWidth(SymId::noteheadBlack) * 0.5;
```

Comment:
Actually I would prefer to apply another solution for it. It looks more like a happy coincidence that adding exactly half of a notehead's width is necessary to resolve the issue for default lyrics font face and size (and score style settings perhaps?). Changing font face or font size with this code results in something like this (the second line is with default settings so is aligned correctly):
![Screenshot_20190611_163032](https://user-images.githubusercontent.com/6000747/59280737-5741e580-8c66-11e9-9029-94abd6c0f77d.png)

If you would still like some solution to be applied perhaps [this commit](https://github.com/dmitrio95/MuseScore/commit/5f00dc3a4c832c5caa65e5ae5662ac0412b8e0d9) could be considered (I mentioned it in the issue's thread). It doesn't work correctly if lyrics font or size is defined via controls available in text editing mode but works correctly for any font face and size provided they are set to lyrics via Inspector.

Reply:
I'm fine either way. Maybe both, and my fix commented out?

Reply:
Well, maybe they could be both, but I really fail to see how the problem of aligning a text line to the given position is related to a notehead width :)

Reply:
Because currenly the start of the letter seems to get aligned to the center to the notehead, rather than to its left edge. Or the center of the letter to the noteheads right edge. 
A few lines later in the code the same method is used, there too the nothead width gets determined

Reply:
> Because currenly the start of the letter seems to get aligned to the center to the notehead, rather than to its left edge. Or the center of the letter to the noteheads right edge.

The experiment with changing font face/size (both via Inspector and text editing panel) seems to prove the opposite.

> A few lines later in the code the same method is used, there too the nothead width gets determined

That code exactly tries to center the lyrics line under the notehead if alignment is set to `HCENTER`. For that task it is indeed necessary to know the notehead width (although I wonder why `noteheadBlack` is used while the actual notehead may differ). The problem which the discussed code is trying to solve is just to determine width of leading and trailing parts of the lyrics to use that in the alignment algorithm later, and those widths do not depend on a notehead size.

Reply:
Well, close this PR then and apply your fix. Main thing is to get this fixed.
The issue might come back though, should that `#if 0 ` ever gets changed...